strongbox-security/strongbox-authentication-registry/src/test/java/org/carlspring/strongbox/authentication/ConfigurableProviderManagerTest.java

```diff
@@ -59,7 +60,7 @@
     private int externalUsersInvalidateSeconds;
 
     @Inject
-    @OrientDb
+    @Database
     private UserService orientDbUserService;
```

Comment:
The name of this field for the service no longer reflects the reality, as we're not using OrientDB. Could you please rename this accordingly? Thanks! :)

Reply:
fixed